packages/ngx-deploy-npm/project.json

```diff
@@ -82,6 +82,20 @@
       },
       "dependsOn": ["build"]
     },
+    "yarn-link-it": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "cd dist/packages/ngx-deploy-npm && npx --yes yarn link"
+      },
+      "dependsOn": ["build"]
```

Comment:
Shouldn't be?

```suggestion
      "dependsOn": ["^build"]
```

Reply:
No. `["^build"]` in this scenario doesn't mean it will build the project dependencies. The dependency building is already taken care in the file `nx.json`

```json
 "targetDefaults": {
    "build": {
      "inputs": ["production", "^production"],
      "dependsOn": ["^build"]
    },
}
```